src/main/ui/webpack.config.js

```diff
@@ -54,7 +54,12 @@ module.exports = function (env) {
         externals: { // JS libraries that we link from a CDN in index.ejs
             'react': 'React',
             'react-dom': 'ReactDOM',
-            'react-router': 'ReactRouter'
+            'react-router': 'ReactRouter',
+            'Config': JSON.stringify(env === "prod" ? {
```

Comment:
@BartlomiejRasztabiga I can see that you haven't defined the variable in the right place i.e. the "external" key within which you defined the variable is to be used only for listing the external libraries that would be required as mentioned in the comment there. You need to define this variable below the 16th line.

Reply:
So when I got this const at 16th line, how can I access it from index.jsx? I can't find that on the Internet.